lib/App.js

```diff
@@ -64,6 +64,10 @@ class App {
     return fs.existsSync(path.join(appPath, 'tsconfig.json'));
   }
 
+  static usesESLint({ appPath }) {
+    return fs.existsSync(path.join(appPath, '.eslintrc.json'));
```

Comment:
Is this the only we file we consider for eslint to be enabled? E.g. `.eslintrc` or `.eslintrc.js` are also valid eslint config files right?

Reply:
Eh yea that might be nicer, the possible config files are:

- `.eslintrc.js`
- `.eslintrc.cjs`
- `.eslintrc.yaml`
- `.eslintrc.yml`
- `.eslintrc.json`
- `package.json` (`eslintConfig` property)

We currently only use the check in one place; where we determine what packages we need to install. This happens right after we created the app (and the `.eslintrc.json` file). So it will currently always work as intended.
However if we were to run ESLint during the build phase we should probably extend the check with the other config file formats or use some other way to check if ESLint is enabled.

Unfortunately we can't just check if ESLint is installed in the project because we call this when installing the packages 🤔